channeldb/channel.go

```diff
@@ -168,6 +168,10 @@ type ChannelConstraints struct {
 	// time lock. Similarly, any HTLC's offered by this node should use
 	// this value as well.
 	CsvDelay uint16
+
+	// NumConfs is the minimum number of confirmations required before the
+	// channel can begin operating between both parties.
+	NumConfs uint32
```

Comment:
must also be serialized in the db?

Reply:
I'd say it doesn't need to be since we don't persist funding requests anyway. The only inconsistency I see is that when reading the `ChannelConfig` from the DB, this will always be 0. We can either leave a `NOTE` detailing so, or enforce the `MaxConfs` check elsewhere.

Reply:
I don't think we should make it part of the `ChannelConstraints` if we don't plan to add it to the database. `ChannelConstraints` are meant for asymmetric values that need to be adhered to during channel operation.

Reply:
Fixed.